DAL/basic_db_functions.py

```diff
@@ -0,0 +1,15 @@
+import random
+from pymongo import MongoClient
+from db.connection_credentials import mongodb_url
+
+
+class BasicDbFunctions:
```

Comment:
is there a special reason to use a class here?

Reply:
I could refrain from using a class here, it’s just a convention I follow, connection with databases is done by classes. That’s the convention I follow and followed for my previous project.